lib/screens/contact_us_page.dart

```diff
@@ -0,0 +1,234 @@
+      child: Text(
+        '店舗情報',
+        style: TextStyle(
+          fontSize: 16.0,
+          fontWeight: FontWeight.bold,
+          color: paletteBlackColor,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildStoreInfoTable() {
+    return Container(
+      decoration: BoxDecoration(border: Border.all(color: kPaletteBorderColor)),
+      child: Column(
+        children: <Widget>[
+          _buildStoreNameRow(),
+          _buildStoreAddressRow(),
+          _buildStoreContactRow(),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildStoreNameRow() {
+    return Container(
+      height: 55.0,
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(color: kPaletteBorderColor),
+        ),
+      ),
+      child: Row(
+        children: <Widget>[
+          Container(
+            width: 100.0,
+            padding: EdgeInsets.only(left: 10.0),
+            alignment: Alignment.centerLeft,
+            decoration: BoxDecoration(
+              color: Color(0xFFE3E3E3),
+              border: Border(
+                right: BorderSide(color: kPaletteBorderColor),
+              ),
+            ),
+            child: Text('ショップ名', style: _tableTextStyle),
+          ),
+          Container(
+            padding: EdgeInsets.only(left: 10.0),
+            alignment: Alignment.centerLeft,
+            child: Text('バディカフェ', style: _tableTextStyle),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildStoreAddressRow() {
+    return Container(
+      height: 95.0,
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(color: kPaletteBorderColor),
+        ),
+      ),
+      child: Row(
+        children: <Widget>[
+          Container(
+            width: 100.0,
+            padding: EdgeInsets.only(left: 10.0),
+            alignment: Alignment.centerLeft,
+            decoration: BoxDecoration(
+              color: Color(0xFFE3E3E3),
+              border: Border(
+                right: BorderSide(color: kPaletteBorderColor),
+              ),
+            ),
+            child: Text('住所', style: _tableTextStyle),
+          ),
+          Expanded(
+            child: Container(
+              padding: EdgeInsets.only(left: 10.0, top: 15.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text('〒444-2149', style: _tableTextStyle),
+                  Text('愛知県岡崎市細川町字窪地77-207', style: _tableTextStyle),
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildStoreContactRow() {
+    return Container(
+      height: 120.0,
+      child: Row(
+        children: <Widget>[
+          Container(
+            width: 100.0,
+            padding: EdgeInsets.only(left: 10.0),
+            alignment: Alignment.centerLeft,
+            decoration: BoxDecoration(
+              color: Color(0xFFE3E3E3),
```

Comment:
since this color is used in multiple places, maybe make a const in the colors.dart file and reference this from there?